third_party/forked/README.md

```diff
@@ -0,0 +1,3 @@
+# rerun-process-wrapper
+
+Forked from https://github.com/windmilleng/rerun-process-wrapper
```

Comment:
Can you reach out to them to see about a LICENSE for this code?

Reply:
Yeah, already done. I suggested Apache 2.0 but we'll see. https://github.com/windmilleng/rerun-process-wrapper/issues/3

Reply:
Thanks!

Reply:
https://github.com/windmilleng/rerun-process-wrapper

added apache 2.0

Reply:
Can you pull in their license file? Maybe also include the git commit that you copied from?

Reply:
:100: